summit.html

```diff
@@ -161,6 +155,25 @@ <h2>Day 1 (Tuesday, June 13)</h2>
                     <p>
                       <b>Speakers:</b> All
                     </p>
+                    <p>
+                      Kristofer Hallén, Ericsson - Reflections on Ericsson's Eiffel usage
+                      <a href="https://youtu.be/y2uEp9ZX8MM">Video</a>
+                    </p>
+                    <p>
+                      Magnus Bäck, Axis Communications - Are multiple contexts a
+                      thing?
+                      <a href="https://github.com/eiffel-community/community/blob/master/presentations/eiffel_summit_2023.1/are_multiple_contexts_a_thing.pdf">Slides</a>
+                      &nbsp;<a href="https://youtu.be/bHLsf9x8ik4">Video</a>
+                    </p>
+                    <p>
+                      Jonathan Dartland, Nasdaq - Eiffel usage at Nasdaq
+                      <a href="https://github.com/eiffel-community/community/blob/master/presentations/eiffel_summit_2023.1/eiffel-at-nasdaq-2023-1.pdf">Slides</a>
+                      &nbsp;<a href="https://youtu.be/kkNlt_7M0Ac">Video</a>
+                    </p>
+                    <p>
+                      Erik Sternerson, Volvo Cars - Confidence Labels
```

Comment:
Nit (in other places too):

```suggestion
                      Erik Sternerson, Volvo Cars &ndash; Confidence Labels
```

Reply:
Corrected in new commit